specs/waku/waku-1.md

```diff
@@ -60,11 +59,11 @@ Waku was created to incrementally improve in areas that Whisper is lacking in, w
 
 ## Definitions
 
-| Term            | Definition                                                                  |
-| --------------- | ----------------------------------------------------------------------------|
-| **Light node**  | A Waku node that does not forward any envelopes through the Messages packet.|
-| **Envelope**    | Messages sent and received by Waku nodes.                                   |
-| **Node**        | Some process that is able to communicate for Waku.                          |
+| Term            | Definition                                                                              |
+| --------------- | ----------------------------------------------------------------------------------------|
+| **Light node**  | A Waku node that does not forward any envelopes through the Messages packet.            |
+| **Envelope**    | Messages sent and received by Waku nodes. Described in [ABNF spec `waku-envelope`](#abnf-specification) |
```

Comment:
I'm unsure about mentioning the abnf spec here. Perhaps we should put something general regarding this in the intro, else we might end up repeating this many more times.

Reply:
I don't have a problem with this, but whatever is implemented needs to make it easy for a reader to navigate directly to the Envelope related information. The main reason I implemented this change is because when I was reading these docs I wanted to go back and refer to the Envelope specific section, and I couldn't find it.

After your comment on Discuss I could see that there was no dedicated section for Envelopes and that it was under the umbrella of the ABNF spec. It wasn't easy for me find this information, and I this change is designed to make it easy for anyone that is looking for Envelope specific information.

Most other items defined in the ABNF spec have a dedicated paragraph explaining a little more about them. I didn't want to change the document too much, so I felt that pointing the reader to the place where Envelope is defined would be more agreeable than introducing a new paragraph.

Reply:
Yeah, I understand the reasoning behind it. Not a big issue for me, could stay also, merely a remark that I rather don't see the spec ending up having links everywhere to the ABNF spec.

Reply:
Ok, I understand. I think that Envelope would be the only item that requires a reference to the ABNF spec, as all other items are given a dedicated section.